### PR TITLE
Add SSP mods configuration templates

### PR DESCRIPTION
**Pull request by nikosev**
This PR contains the following changes:
- Update config templates for ssp mods and move them to ssp-modules role (metarefresh, cron, discopower)
- Add config templates for ssp mods in ssp-modules role (rciaminfo, themevanilla,proxystatistics)
- Update checks for metarefresh in `ssp:config:config` tag of SSP role
- Disable checks for `ssp:config:mods` tag in SSP role for SSP versions '1.18' or greater
- Add tags in ssp-module role

### CODE REVIEW

File: roles/ssp/tasks/configure-common.yml [hunk 1]
```diff
@@ -3,6 +3,13 @@
 ---
 
 
+- name: Check if SSP mod metarefresh is enabled
+  set_fact:
+    ssp_mod_metarefresh_enabled: true
+  when: ssp_mods_extra_enabled | selectattr('name','equalto','metarefresh') | list | count > 0
+  tags:
+    - ssp:config:config
+
```
Comment (NicolasLiampotis) [inline roles/ssp/tasks/configure-common.yml:12]: Remove if not needed anymore

File: roles/ssp/tasks/configure-common.yml [hunk 2]
```diff
@@ -200,15 +207,15 @@
 - name: Ensure required SSP metarefresh module dirs exist
   file: path={{ ssp_metadatadir }}/{{ item.output_dir }} state=directory owner={{ssp_webuser }} group={{ ssp_webgroup }}
   with_items: "{{ ssp_mod_metarefresh_sets }}"
-  when: "'metarefresh' in ssp_mods_enabled"
+  when: "'metarefresh' in ssp_mods_enabled or ssp_mods_extra_enabled | selectattr('name','equalto','metarefresh') | list | count > 0"
```
Comment (NicolasLiampotis) [inline roles/ssp/tasks/configure-common.yml:210]: Remove if not needed anymore

File: roles/ssp/tasks/configure-common.yml [hunk 3]
```diff
@@ -200,15 +207,15 @@
...
   become: yes
   tags:
     - ssp:config:mods
     - ssp:config:mods:metarefresh
 
 - name: Configure SSP metarefresh module
   template:
-    src: "config/config-metarefresh-{{ ssp_version }}.php.j2"
+    src: "config/config-metarefresh.php.j2"
```
Comment (NicolasLiampotis) [inline roles/ssp/tasks/configure-common.yml:218]: Revert change

File: roles/ssp/templates/config/config-1.18.php.j2 [hunk 4]
```diff
@@ -1111,7 +1111,7 @@ $config = [
      */
     'metadata.sources' => [
         ['type' => 'flatfile'],
-{% if 'metarefresh' in ssp_mods_enabled %}
+{% if 'metarefresh' in ssp_mods_enabled or ssp_mod_metarefresh_enabled is defined and ssp_mod_metarefresh_enabled is true %}
```
Comment (NicolasLiampotis) [inline roles/ssp/templates/config/config-1.18.php.j2:1114]: Remove if not needed anymore

File: roles/ssp/templates/config/config-metarefresh.php.j2 [hunk 5]
```diff
@@ -1,19 +1,20 @@
 <?php
```
Comment (NicolasLiampotis) [inline roles/ssp/templates/config/config-metarefresh.php.j2:1]: Rename to 1.18

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: roles/ssp/tasks/configure-common.yml (hunk 2 above)
Comment (nikosev) [reply roles/ssp/tasks/configure-common.yml]: done

File: roles/ssp/tasks/configure-common.yml (hunk 3 above)
Comment (nikosev) [reply roles/ssp/tasks/configure-common.yml]: done

File: roles/ssp/templates/config/config-1.18.php.j2 (hunk 4 above)
Comment (nikosev) [reply roles/ssp/templates/config/config-1.18.php.j2]: done

File: roles/ssp/templates/config/config-metarefresh.php.j2 (hunk 5 above)
Comment (nikosev) [reply roles/ssp/templates/config/config-metarefresh.php.j2]: done

File: roles/ssp/templates/config/module_cron.php.j2 [hunk 6]
```diff
@@ -1,18 +1,19 @@
...
 {% for entry in ssp_mod_cron_entries %}
         '{{ entry.tag }}',
 {% endfor %}
-    ),
+    ],
     'debug_message' => false,
     'sendemail' => false,
 
-);
+];
```
[the remark this answers is not in the document]
Comment (nikosev) [reply roles/ssp/templates/config/module_cron.php.j2]: done

File: roles/ssp/templates/config/module_discopower.php.j2 [hunk 7]
```diff
@@ -1,9 +1,10 @@
 <?php
+
 /* 
  * Configuration for the DiscoPower module.
  */
 
-$config = array (
+$config = [
```
[the remark this answers is not in the document]
Comment (nikosev) [reply roles/ssp/templates/config/module_discopower.php.j2]: done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**